torch/csrc/jit/script/function_schema_parser.cpp

```diff
@@ -58,7 +59,9 @@ struct SchemaParser {
     });
     idx = 0;
     L.expect(TK_ARROW);
-    if (L.cur().kind == '(') {
+    if (L.nextIf(TK_DOTS)) {
+      is_varret = true;
+    } else if (L.cur().kind == '(') {
       parseList('(', ',', ')', [&] {
         returns.push_back(
```

Comment:
for completeness, we should support parsing `...` here too (just like above for the input args)